src/auth/jwt-interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface JwtPayload {
+  userId: string;
+  email: string;
```

Comment:
PullProofAI

The `JwtPayload` interface in `jwt-interface.ts` defines the structure of the JWT payload. It includes `userId` and `email` properties. This interface seems well-defined and aligns with the typical information stored in a JWT payload.

This change looks good as it clearly specifies the shape of the JWT payload, providing clarity on the expected properties within the token.


<!-- This is an auto-generated comment by PullProofAI -->